src/main/java/zd/zero/waifu/motivator/plugin/settings/WaifuMotivatorState.kt

```diff
@@ -0,0 +1,22 @@
+package zd.zero.waifu.motivator.plugin.settings
+
+class WaifuMotivatorState {
+
+    var isWaifuOfTheDayEnabled = true
+
+    var isStartupMotivationEnabled = true
+
+    var isStartupMotivationSoundEnabled = true
+
+    var isUnitTesterMotivationEnabled = true
+
+    var isUnitTesterMotivationSoundEnabled = true
+
+    var isMotivateMeEnabled = true
+
+    var isMotivateMeSoundEnabled = true
+
+    var isSayonaraEnabled = true
+
+    var disableInDistractionFreeMode = true
```

Comment:
It bothers me that `disableInDistractionFreeMode` is not `isDisableInDistractionFreeMode`. 1.2 has not been released to users right? Can this be changed?

Reply:
We can change this one to `isDisabledInDistractionFreeMode`. By changing this, will we have `this.state.isDisabledInDistractionFreeMode()` instead of `this.state.getDisableInDistractionFreeMode()`?

Reply:
Yes! [Here is the docs on that.](https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#properties)